components/Sidebar/SidebarParts.tsx

```
@@ -46,16 +49,45 @@ export const Sidebar = React.forwardRef<
+                onClick={() => setOpenMobile(true)}
+                aria-label="Open Sidebar"
+                title="Open Sidebar"
+              >
+                <PanelLeft className="h-4 w-4" />
+              </button>
+              <button
+                className="inline-flex h-9 w-9 items-center justify-center rounded-md text-sm transition-colors hover:bg-accent hover:text-accent-foreground"
+                onClick={() => router.push("/chat")}
+                aria-label="New Chat"
+                title="New Chat"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)